simplq/src/components/pages/Status/StatusContainer.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { CircularProgress } from '@material-ui/core';
+import { setJoinerStep } from '../../../store/appSlice';
+import styles from '../../../styles/statusPage.module.scss';
+
+export default (props) => {
+  const dispatch = useDispatch();
+
+  let status = null;
+  if (props.updateInProgress) {
+    status = null;
+  } else if (props.tokenStatus === 'REMOVED') {
+    status = <p>You have been removed from the queue</p>;
+  } else if (props.tokenStatus === 'NOTIFIED') {
+    dispatch(setJoinerStep(3));
+    status = <p>Your turn is up </p>;
+  } else if (props.aheadCount === 0) {
+    status = <p>There is no one ahead of you. Please wait to be notified by the queue manager.</p>;
+  } else {
+    /* eslint-disable react/jsx-one-expression-per-line */
+    status = (
+      <p>
+        People ahead of you :<span className={styles.count}>{props.aheadCount}</span>
+      </p>
```

Comment:
Lets instead move the span to the next line

Reply:
Prettier makes it a single line and eslint throws an error saying it should be in different lines. I'll try configuring prettier to handle this.

Reply:
ah ok. I don't know if prettier will leave it alone if we do something like this?
```
 <p>
        <span>People ahead of you :</span>
        <span className={styles.count}>{props.aheadCount}</span>
 </p>
```